packages/broker/plugins.md

```diff
@@ -1,17 +1,34 @@
 # Plugins
 
+The Broker ships with a number of plugins that add functionality or APIs.
+
 ## Table of Contents
-- [Dependencies](#dependencies)
+- [Authentication](#authentication)
 - [Websocket](#websocket)
 - [MQTT](#mqtt)
 - [PublishHttp](#publishhttp)
```

Comment:
I guess it's a bit late now, but can we call the plugin `HTTP` rather than `PublishHttp` or is there good reason behind the name?

Reply:
I guess the reason for the name is that it only supports publish (subscribing over HTTP is non-trivial, as HTTP is a request-response protocol). That being said, I don't see a problem in renaming it to `http` and just saying in the docs that it only supports publish for now.

I did a quick cleanup: In `plugins.md` I used "HTTP" instead of `publishHttp` where possible, however changing the actual plugin name needs to be left for [another ticket](https://linear.app/streamr/issue/NET-820/rename-publishhttp-plugin-to-http).

Then there's the interesting topic of how to actually enable subscribing over HTTP, and is that even necessary. There could be a pollable endpoint that returns the latest message, but it's unclear how subscribing and unsubscribing works, and getting the latest message doesn't fulfill all use cases. HTTP streaming responses are possible too (maybe over SSE API?), but not as easy to use as simple request/response.